apollo-ios/Package.swift

```diff
@@ -64,6 +64,7 @@ let package = Package(
           description: "Installs the Apollo iOS Command line interface."),
         permissions: [
           .writeToPackageDirectory(reason: "Creates a symbolic link to the CLI executable in your project directory."),
```

Comment:
Is this permission still relevant? It looks like we download the binary directly to the project folder now instead of creating a symbolic link.

On second though - we may still need the permission to write to the directory but the reason text should probably be updated then.

Reply:
Ah yea good call out, I can get that updated.